packages/cozy-stack-client/src/TriggerCollection.js

```diff
@@ -28,8 +29,37 @@ class TriggerCollection {
     this.stackClient = stackClient
   }
 
-  async all() {
-    throw new Error('all() method is not yet implemented')
+  /**
+   * Get the list of triggers.
+   *
+   * @see https://docs.cozy.io/en/cozy-stack/jobs/#get-jobstriggers
+   * @param  {{Worker}} options The fetch options: Worker allow to filter only triggers associated with a specific worker.
+   * @return {{data}} The JSON API conformant response.
+   * @throws {FetchError}
+   */
+  async all(options = {}) {
+    const url = uri`/jobs/triggers`
+    const path = querystring.buildURL(url, options)
+    const defaultJSONAPI = {
+      next: false,
+      skip: 0
+    }
+
+    try {
+      const resp = await this.stackClient.fetchJSON('GET', path)
+
+      return {
+        data: resp.data.map(row => normalizeDoc(row, TRIGGERS_DOCTYPE)),
+        meta: { count: resp.data.length },
+        ...defaultJSONAPI
+      }
+    } catch (error) {
+      if (error.message.match(/not_found/)) {
+        return { data: [], meta: { count: 0 }, ...defaultJSONAPI }
```

Comment:
Not blocking here, but just noticed it: we have the same kind of structure [here](https://github.com/cozy/cozy-client/blob/70c98872ba98bddebb85cd1b21a3053e8a11de4d/packages/cozy-stack-client/src/DocumentCollection.js#L55), [here](https://github.com/cozy/cozy-client/blob/70c98872ba98bddebb85cd1b21a3053e8a11de4d/packages/cozy-stack-client/src/DocumentCollection.js#L111) and [here](https://github.com/cozy/cozy-client/blob/70c98872ba98bddebb85cd1b21a3053e8a11de4d/packages/cozy-stack-client/src/DocumentCollection.js#L157), so we must extract it to a helper. I'm going to open an issue so we don't forget it.

Reply:
I create [a commit for this refactor](https://github.com/cozy/cozy-client/pull/389/commits/5befa2e69549a9b286b6a60fef87fba49a415815).